server/src/main/java/thesistrack/ls1/service/MailingService.java

```diff
@@ -266,16 +266,21 @@ private Multipart createApplicationFilesMailContent(String text, Application app
         messageBodyPart.setContent(text, "text/html; charset=utf-8");
         multipart.addBodyPart(messageBodyPart);
 
-        MimeBodyPart examinationReportAttachment = new MimeBodyPart();
-        examinationReportAttachment.attachFile(uploadService.load(application.getUser().getExaminationFilename()).getFile());
-        multipart.addBodyPart(examinationReportAttachment);
+        String cvFilename = application.getUser().getCvFilename();
+        if (cvFilename != null && !cvFilename.isBlank()) {
+            MimeBodyPart cvAttachment = new MimeBodyPart();
+            cvAttachment.attachFile(uploadService.load(cvFilename).getFile());
+            multipart.addBodyPart(cvAttachment);
+        }
 
-        MimeBodyPart cvAttachment = new MimeBodyPart();
-        cvAttachment.attachFile(uploadService.load(application.getUser().getCvFilename()).getFile());
-        multipart.addBodyPart(cvAttachment);
+        String examinationReportFilename = application.getUser().getExaminationFilename();
+        if (examinationReportFilename != null && !examinationReportFilename.isBlank()) {
+            MimeBodyPart examinationReportAttachment = new MimeBodyPart();
+            examinationReportAttachment.attachFile(uploadService.load(examinationReportFilename).getFile());
+            multipart.addBodyPart(examinationReportAttachment);
```

Comment:
Similar to line 271/272/273
You might consider extracting this to a shared function.

Reply:
good suggestion